File: JavaScript/6kyu/FindTheParityOutlier.js
function findOutlier(integers) {
  var odd = integers.filter((i) => i % 2 != 0);
  var even = integers.filter((i) => i % 2 == 0);

  return odd.length == 1 ? odd[0] : even[0];
}

console.log(findOutlier([1, 1, 0, 1, 1]));

File: CSharp/6kyu/FindTheParityOutlier.cs
using System.Linq;
using System;

namespace CSharp._6Kyu
{
    public static partial class Kata6
    {
        public static int FindOutlier(int[] integers)
        {
            var odd = integers.Where(i => i % 2 != 0).ToList();
            var even = integers.Where(i => i % 2 == 0).ToList();

            return odd.Count() == 1 ? odd.First() : even.First();
        }
    }
}